packages/perseus/src/widgets/label-image/score-label-image.ts

```diff
@@ -43,15 +43,21 @@ export function scoreMarker(
     return score;
 }
 
-// TODO(LEMS-2440): May need to pull answers out of PerseusLabelImageWidgetOptions[markers] for the rubric
 function scoreLabelImage(
     userInput: PerseusLabelImageUserInput,
-    rubric?: PerseusLabelImageRubric,
+    scoringData: PerseusLabelImageScoringData,
 ): PerseusScore {
     let numAnswered = 0;
     let numCorrect = 0;
+    const combinedData = userInput.markers.map((marker, index) => {
+        return {
+            ...marker,
+            answers: scoringData.markers[index].answers,
+        };
+    });
+    const allMarkerData = {markers: combinedData};
 
-    for (const marker of userInput.markers) {
+    for (const marker of allMarkerData.markers) {
```

Comment:
Data was combined here so that `marker` can have all the available information as `scoreMarker` requires an object of `InteractiveMarkerType`, which includes answers.

Reply:
No action needed, but my gut feeling is that this is needless misdirection. I kind of feel we should just change `scoreMarker`:

``` TS
function scoreMarker(
    userInput: APieceOfUserInputType,
    scoringData: APieceOfScoringDataType,
): InteractiveMarkerScore;
```

I'm fine punting on this though.

Reply:
Fixed!